### PR TITLE
feat(n8n Google My Business Node): New node

### PR DESCRIPTION
**Pull request by valentina98**
## Summary

Adds Google My Business node and trigger node

## Related Linear tickets, Github issues, and Community forum posts
https://linear.app/n8n/issue/NODE-1614/community-pr-wip-add-google-my-business-node

https://community.n8n.io/t/google-my-business-node-request/8248/10

### CODE REVIEW

**Comment by Joffcom** [comment]
Hey @valentina98,

Thanks for the PR, We have created "GHC-178" as the internal reference to get this reviewed.

One of us will be in touch if there are any changes needed, in most cases this is normally within a couple of weeks but it depends on the current workload of the team.

File: packages/nodes-base/nodes/Google/MyBusiness/PostDescription.ts [hunk 1]
```diff
@@ -0,0 +1,1083 @@
...
+				routing: {
+					request: {
+						method: 'DELETE',
+						url: '=/{{$parameter["post"]}}',
+						ignoreHttpStatusErrors: true,
+					},
+					output: {
+						postReceive: [
+							async function (
+								this: IExecuteSingleFunctions,
```
Comment (ShireenMissi) [inline packages/nodes-base/nodes/Google/MyBusiness/PostDescription.ts:50]: Can you extract this to GenericFunctions?

File: packages/nodes-base/nodes/Google/MyBusiness/PostDescription.ts [hunk 2]
```diff
@@ -0,0 +1,1083 @@
...
+				description: 'Retrieve details of a specific post',
+				routing: {
+					request: {
+						method: 'GET',
+						url: '=/{{$parameter["post"]}}',
+						ignoreHttpStatusErrors: true,
+					},
+					output: {
+						postReceive: [
+							async function (
```
Comment (ShireenMissi) [inline packages/nodes-base/nodes/Google/MyBusiness/PostDescription.ts:88]: Can you extract this to GenericFunctions?

File: packages/nodes-base/nodes/Google/MyBusiness/PostDescription.ts [hunk 3]
```diff
@@ -0,0 +1,1083 @@
...
+					},
+					request: {
+						method: 'PATCH',
+						url: '=/{{$parameter["post"]}}',
+						ignoreHttpStatusErrors: true,
+					},
+					output: {
+						postReceive: [
+							async function (
+								this: IExecuteSingleFunctions,
```
Comment (ShireenMissi) [inline packages/nodes-base/nodes/Google/MyBusiness/PostDescription.ts:153]: Can you extract this to GenericFunctions?

**Review by ShireenMissi — COMMENTED** [review]
Thanks @ @valentina98 for the PR 🙏
the code is easy to read and very organised 👍
left some small refactoring comments apart from that looks good 🎉

File: packages/nodes-base/nodes/Google/MyBusiness/GoogleMyBusinessTrigger.node.ts [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+			searchLocations,
+		},
+	};
+
+	async poll(this: IPollFunctions): Promise<INodeExecutionData[][] | null> {
+		const nodeStaticData = this.getWorkflowStaticData('node');
+		let responseData;
+		const qs: IDataObject = {};
+
+		// const event = this.getNodeParameter('event') as string; // Currently there is only one event
```
[the remark this answers is not in the document]
Comment (valentina98) [reply packages/nodes-base/nodes/Google/MyBusiness/GoogleMyBusinessTrigger.node.ts]: It is not needed. I have removed it.

File: packages/nodes-base/nodes/Google/MyBusiness/PostDescription.ts (hunk 1 above)
Comment (valentina98) [reply packages/nodes-base/nodes/Google/MyBusiness/PostDescription.ts]: I have extracted it.

File: packages/nodes-base/nodes/Google/MyBusiness/PostDescription.ts (hunk 2 above)
Comment (valentina98) [reply packages/nodes-base/nodes/Google/MyBusiness/PostDescription.ts]: I have extracted it.

**Review by ShireenMissi — APPROVED** [review]
LGTM 🎉

**Comment by janober** [comment]
Got released with [`n8n@1.65.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.65.0)

**Pull request merged**